.github/workflows/linux-ci.yml

```diff
@@ -49,7 +46,17 @@ jobs:
         mkdir -p build && cd build
         cmake -DCMAKE_BUILD_TYPE=Debug \
           -DCMAKE_PREFIX_PATH="${GITHUB_WORKSPACE}/build_deps/prefix;${GITHUB_WORKSPACE}/build_deps/PhysX/physx/install" \
-          -DPython_ADDITIONAL_VERSIONS=2.7 -DPLASMA_BUILD_TESTS=ON \
-          -DPLASMA_BUILD_TOOLS=ON -DPLASMA_BUILD_RESOURCE_DAT=ON ..
+          -DPLASMA_BUILD_TESTS=ON -DPLASMA_BUILD_TOOLS=ON -DPLASMA_BUILD_RESOURCE_DAT=ON ..
         make -j2
+
+    - name: Test Plasma
+      run: |
+        sudo apt-get install xvfb gnome-keyring dbus-x11 libsecret-tools
+        export "$(dbus-launch)"
+        echo "" | gnome-keyring-daemon --unlock
+        gnome-keyring-daemon --start --daemonize --components=secrets
+        export "$(echo "" | gnome-keyring-daemon -r -d --unlock)"
+        echo -n "secret-tool-password-1" | secret-tool store --label=secret-tool-label-1 secret-tool-key-1 secret-tool-value-1
```

Comment:
I tried having all this junk as its own "Setup Test Environment" step, but it doesn't work because the exported env vars don't carry to other steps